config_test.go

```diff
@@ -30,6 +30,23 @@ func TestSwampConfig_ValidateRoleAndAccount(t *testing.T) {
 	assert.Error(t, c.Validate())
 }
 
+func TestSwampConfig_ValidateMFADeviceAndTokenCommand(t *testing.T) {
+	c := NewSwampConfig()
+	c.targetRole = "arn:aws:iam::1234567890:role/some-role"
+	c.tokenSerialNumber = "someSerialNumber"
+	c.mfaExec = "some command"
+
+	assert.NoError(t, c.Validate())
+}
+
+func TestSwampConfig_ValidateNoMFADeviceButTokenCommand(t *testing.T) {
+	c := NewSwampConfig()
+	c.targetRole = "arn:aws:iam::1234567890:role/some-role"
+	c.mfaExec = "some command"
+
+	assert.Error(t, c.Validate())
+}
+
```

Comment:
please add a test case which actually executes a shell command and validates it.

possible test inputs:
* echo `123456`
* echo `123456\n`
* echo `1234567`
* echo `123`